gender_novels/analysis/analysis.py

```diff
@@ -213,27 +213,40 @@ def dunn_individual_word(total_words_m_corpus, total_words_f_corpus, wordcount_f
 
     '''
 
-    #function implementation
-    e1 = total_words_m_corpus * (wordcount_male + wordcount_female) / (total_words_m_corpus +
-                                                                      total_words_f_corpus)
-    e2 = total_words_m_corpus * (wordcount_male + wordcount_female) / (total_words_m_corpus
-                                                                       +total_words_f_corpus)
-    # print("e1 valaue: ", e1)
-    # print("e2 value: ", e2)
+def dunn_individual_word(total_words_corpus_1, total_words_corpus_2, count_of_word_corpus_1,
+                     count_of_word_corpus_2):
+    '''
+    applies dunning log likelihood to compare individual word usage in male and female corpus
 
-    dunning_log_likelihood = 2 * (wordcount_male * math.log(wordcount_male / e1)) +2*(
-        wordcount_female*math.log(wordcount_female/e2))
+    :param word: desired word to compare
+    :param m_corpus: c.filter_by_gender('male')
+    :param f_corpus: c. filter_by_gender('female')
+    :return: log likelihoods and p value
+    >>> total_words_m_corpus = 8648489
+    >>> total_words_f_corpus = 8700765
+    >>> wordcount_female = 1000
+    >>> wordcount_male = 50
+    >>> dunn_individual_word(total_words_m_corpus,total_words_f_corpus,wordcount_male,wordcount_female)
 
-    # print(math.log10(wordcount_male / e1))  #WHY IS THIS VALUE ZERO??
-    # print(math.log10(wordcount_female / e2))  #WHY IS THIS VALUE ZERO??
+    '''
+    a = count_of_word_corpus_1
+    b = count_of_word_corpus_2
+    c = total_words_corpus_1
+    d = total_words_corpus_2
```

Comment:
Are there other names you could use for these variables that are a bit more descriptive?

Reply:
In this rare case, using the single letter variables is fine because it makes it much clearer what you do with the values in the following steps :
`e1 = c * (a + b) / (c + d)` 
is clearer than
`e1 = total_words_corpus_1 * (count_of_word_corpus_1 + count_of_words_corpus_2) / ...`